sensor/common/scan/scan.go

```diff
@@ -270,10 +348,17 @@ func (s *LocalScan) fetchSignatures(ctx context.Context, errorList *errorhelpers
 	// Fetch signatures from cluster-local registry.
 	sigs, err := s.fetchSignaturesWithRetry(ctx, signatures.NewSignatureFetcher(), image, image.GetName().GetFullName(), registry)
 	if err != nil {
-		log.Debugf("Failed fetching signatures for image %q: %v", image.GetName().GetFullName(), err)
+		// Like Central, only log errors related to fetching signatures.
+		if !errors.Is(err, errox.NotAuthorized) {
+			log.Errorf("Fetching image signatures for image %q: %v", image.GetName().GetFullName(), err)
+		} else {
+			// Log not authorized errors in debug mode, since we expect them to occur.
+			log.Debugf("Unauthorized error fetching image signatures for image %q: %v", image.GetName().GetFullName(), err)
+		}
+	}
+
+	if len(sigs) == 0 {
```

Comment:
before we assumed only errors meant no signatures, but we have found that to be false, correct?